### PR TITLE
fixture support for virt-who config cli : data_form deploy_type virtwho_config delete_host& manifest content move

### PR DESCRIPTION
**Pull request by yanpliu**
Tasks:
1. fixture support for virt-who config cli : data_form deploy_type virtwho_config delete_host
2. module_sca_manifest content move because module_sca_manifest.content can not found

Cases PASS
```
(robottlo_vv311) [virtwho@dell-per740-68-vm-04 robottelo]$ pytest ./tests/foreman/virtwho/cli/test_esx.py  --disable-pytest-warnings -q
............                                                                                                                                                                                                [100%]

12 passed, 12 deselected, 7 warnings in 1140.02s (0:19:00)
(robottelo_vv_311_master) [virtwho@dell-per740-68-vm-04 robottelo]$ pytest ./tests/foreman/virtwho/cli/test_esx_sca.py --disable-pytest-warnings -q
................                                                                                                                                                                                            [100%]
16 passed, 16 deselected, 7 warnings in 2903.38s (0:48:23)

(robottelo_vv_311_master) [virtwho@dell-per740-68-vm-04 robottelo]$ pytest ./tests/foreman/virtwho/cli/test_hyperv.py --disable-pytest-warnings -q
...                                                                                                                                                                                                         [100%]
3 passed, 3 deselected in 574.06s (0:09:34)

(robottelo_vv_311_master) [virtwho@dell-per740-68-vm-04 robottelo]$ pytest ./tests/foreman/virtwho/cli/test_hyperv_sca.py --disable-pytest-warnings -q
...                                                                                                                                                                                                         [100%]
3 passed, 3 deselected in 539.66s (0:08:59)

(robottelo_vv_311_master) [virtwho@dell-per740-68-vm-04 robottelo]$ pytest ./tests/foreman/virtwho/cli/test_libvirt**.py --disable-pytest-warnings -q
......                                                                                                                                                                                                      [100%]
6 passed, 6 deselected, 5 warnings in 622.06s (0:10:22)

(robottelo_vv_311_master) [virtwho@dell-per740-68-vm-04 robottelo]$ pytest ./tests/foreman/virtwho/cli/test_kubevirt**.py --disable-pytest-warnings -q
......                                                                                                                                                                                                      [100%]
6 passed, 6 deselected, 5 warnings in 832.63s (0:13:52)
(robottelo_vv_311_master) [virtwho@dell-per740-68-vm-04 robottelo]$ pytest ./tests/foreman/virtwho/cli/test_nutanix**.py --disable-pytest-warnings -q
.............                                                                                                                                                                                               [100%]
13 passed, 13 deselected, 5 warnings in 1341.35s (0:22:21)
```

### CODE REVIEW

**Review by JacobCallahan — CHANGES_REQUESTED** [review]
ACK on content, but can you rebase this and run pre-commit on these files?

**Comment by yanpliu** [comment]
> ACK on content, but can you rebase this and run pre-commit on these files?

@JacobCallahan Hello Jack, have already rebased the fix the conflict.

**Review by JacobCallahan — APPROVED** [review]

File: pytest_fixtures/component/taxonomy.py [hunk 1]
```diff
@@ -99,7 +99,7 @@ def module_entitlement_manifest_org(module_org, module_entitlement_manifest, mod
 @pytest.fixture(scope='module')
 def module_sca_manifest_org(module_org, module_sca_manifest, module_target_sat):
     """Creates an organization and uploads an SCA mode manifest generated with manifester"""
-    module_target_sat.upload_manifest(module_org.id, module_sca_manifest.content)
+    module_target_sat.upload_manifest(module_org.id, module_sca_manifest)
```
Comment (shubhamsg199) [inline pytest_fixtures/component/taxonomy.py:102]: This has already been fixed, can you revert it?

**Review by shubhamsg199 — CHANGES_REQUESTED** [review]

**Comment by Gauravtalreja1** [comment]
@yanpliu Try rebase and `pre-commit autoupdate; pre-commit run --all-files` locally to fix pre-commit checks here.

**Comment by yanpliu** [comment]
> @yanpliu Try rebase and `pre-commit autoupdate; pre-commit run --all-files` locally to fix pre-commit checks here.

Thank you very much Gaurav, have already fixed, could you please review and merge the PR? Thanks

**Comment by yanpliu** [comment]
Hello, can you help to review and merge this PR? Thank you very much . As satellite6.15 is coming ,this PR influence most of the cases, thank you very much

**Comment by yanpliu** [comment]
@Gauravtalreja1 Could you please help to review and merge this PR? because this PR has influcence to PRs have been merged already. #12640 #12620 . And can this PR can merge to satellite6.15? Shall I need to submit these changes on satellite6.15?

**Review by Gauravtalreja1 — APPROVED** [review]

**Pull request merged**